test/integration/fixtures/simple-reporter.fixture.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+var Base = require('../../../lib/reporters/base');
+var constants = require('../../../lib/runner').constants;
+var RUNNER_EVENT_PASS = constants.RUNNER_EVENT_PASS;
+var RUNNER_EVENT_FAIL = constants.RUNNER_EVENT_FAIL;
+var RUNNER_EVENT_SUITE = constants.RUNNER_EVENT_SUITE;
+var RUNNER_EVENT_END = constants.RUNNER_EVENT_END;
+var RUNNER_EVENT_TEST_END = constants.RUNNER_EVENT_TEST_END;
+
+module.exports = SimpleReporter;
+
+function SimpleReporter(runner) {
+  Base.call(this, runner);
+
+  runner.on(RUNNER_EVENT_SUITE, function(suite) {
+    console.log("on('suite') called");
```

Comment:
Why aren't the event names being used in console messages?

Reply:
You mean the constants?  What the reporter actually does here is irrelevant to the existing integration tests, which just check that the reporter can load without error.

Reply:
I'm going to replace this with the "example" reporter.

Reply:
```js
  runner.on(RUNNER_EVENT_SUITE, function(suite) {
    console.log("on('" + RUNNER_EVENT_SUITE + "') called");
  });
```